habitat/config/default.py

```diff
@@ -367,3 +376,31 @@ def get_config(
 
     config.freeze()
     return config
+
+
+def get_multi_task_config(
+    config_paths: Optional[Union[List[str], str]] = None,
+    opts: Optional[list] = None,
+) -> CN:
+    config = get_config(config_paths, opts)
+    if len(config.TASKS):
```

Comment:
Can a config.TASKS config be read by the normal config loader? Should it error? @erikwijmans thoughts?

Reply:
Yep, it can be.  I think merging this function will solve this concern.